lms/djangoapps/course_api/blocks/transformers/block_completion.py

```diff
@@ -0,0 +1,86 @@
+"""
+Block Completion Transformer
+"""
+
+from xblock.completable import XBlockCompletionMode as CompletionMode
+
+from lms.djangoapps.completion.models import BlockCompletion
+from openedx.core.djangoapps.content.block_structure.transformer import BlockStructureTransformer
+
+
+class BlockCompletionTransformer(BlockStructureTransformer):
+    """
+    Keep track of the completion of each block within the block structure.
+    """
+    WRITE_VERSION = 1
+    READ_VERSION = 1
+    COMPLETION = 'completion'
+
+    def __init__(self):
+        pass
```

Comment:
Are you deliberately trying to suppress the behavior of the superclass's `__init__()` method, or is this just vestigial code?   This could either use an explanatory comment, or should be deleted.

Reply:
Well spotted :-) Actually superclass doesn't define `__init__` and inherits itself from `object` but I'll add a call to superclass's constructor.

Reply:
If you're not doing anything else, just delete the definition altogether.  If you leave it out, it will search the superclasses automatically.